### PR TITLE
Updated Medication Order DSTU2 Resource

### PR DESCRIPTION
**Pull request by kckeim**
Had various formatting changes on this one to get it into a similar style to other resources that were recently updated.
Verified/updated links throughout document.
Validated that API calls behave as expected with both the expected and unexpected parameters.

Description
----
- Uppercasing and punctuation adds throughout document.
- Many links to HL7.org updated to use https.
  - Also fixed dead links for ICD terminologies.
- Updated Overview Paragraph.
- Consolidated Disclaimer + Errors & Outcomes links.
- Clarified Custom Extension.
- Updated Parameter Table + Implementation Notes to match current standards, and behavior of the API.
*****************************
PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [X] Screenshot(s) of changes attached after changes merged and published.

****************************
Old Overview:
image

New Overview:
image

****************************
Old Custom Extensions:
image

New Custom Extensions:
image

****************************
Old Notes + Parameter Table:
image
image

New Parameter Table + Notes:
image
image
****************************

### CODE REVIEW

**Review by RicardoCrespo-ora — APPROVED** [review]

**Review by cldeitrich — APPROVED** [review]
[PRODOC-203606](https://jira.cerner.com/browse/PRODOC-203606): Approved for style.

File: content/millennium/dstu2/medications/medication-order.md [hunk 1]
```diff
@@ -53,42 +59,49 @@ The following fields are returned if valued:
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2/StructureDefinition/{id}`
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-order.md:59]:
```suggestion
All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2/StructureDefinition/{id}`
For more information, see the [`StructureDefinition`] resource.
```

File: content/millennium/dstu2/medications/medication-order.md [hunk 2]
```diff
@@ -136,20 +143,16 @@ List an individual MedicationOrder by its id:
...
+[`_count`]: https://hl7.org/fhir/dstu2/search.html#count
+[`number`]: https://hl7.org/fhir/dstu2/search.html#number
+[contained]: https://hl7.org/fhir/DSTU2/references.html#contained
 [Patient friendly display]: #custom-extensions
-[`string`]: http://hl7.org/fhir/DSTU2/datatypes.html#string
+[`string`]: https://hl7.org/fhir/DSTU2/datatypes.html#string
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
+[CodeableConcept]: https://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
+[status]: https://www.hl7.org/fhir/dstu2/valueset-medication-order-status.html
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-order.md:158]:
```suggestion
[status]: https://www.hl7.org/fhir/dstu2/valueset-medication-order-status.html
[`StructureDefinition`]: ../../conformance/structure-definition/#overview
```

**Review by amcginn — APPROVED** [review]

**Comment by shushruth-s** [comment]
After Deployment:
Screenshot 2024-02-02 at 3 17 34 PM
Screenshot 2024-02-02 at 3 18 07 PM
Screenshot 2024-02-02 at 3 18 41 PM
Screenshot 2024-02-02 at 3 19 01 PM

**Pull request merged**